Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System.ComponentModel;
+using System.Windows;
+using System.Windows.Media;
+using WThickness = System.Windows.Thickness;
+
+namespace Xamarin.Forms.Platform.WPF
+{
+	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
+	{
+		bool _fontApplied;
+		FormsRadioButton _button;
+
+		public bool IsDisposed { get; private set; }
```

Comment:
Is a public property necessary here? If it is not, change it to use a bool variable bool `_isDisposed`.

Reply:
Done